### PR TITLE
docs/md: fix unresolved reference

### PR DESCRIPTION
**Pull request by isoppp**
NewSchema caused an unresolved reference error when I was proceeding with the tutorial. This change was necessary to run the application.

ref: https://github.com/a8m/ent-graphql-example/blob/master/cmd/todo/main.go#L38

### CODE REVIEW

File: doc/md/tutorial-todo-gql.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ func main() {
 	}
 
 	// Configure the server and start listening on :8081.
-	srv := handler.NewDefaultServer(NewSchema(client))
+	srv := handler.NewDefaultServer(todo.NewSchema(client))
```
Comment (a8m) [inline doc/md/tutorial-todo-gql.md:264]: Thanks for fixing this @isoppp.

I think the `"todo"` import is also necessary. Can you add it to this PR? 🙏

Comment (isoppp) [reply doc/md/tutorial-todo-gql.md:265]: Ah, thank you for pointing it out, it's also necessary.
Fixed it 🙏

**Review by a8m — APPROVED** [review]
Thanks for your contribution @isoppp ❤️

**Comment by a8m** [comment]
@all-contributors please add @isoppp for docs.

**Pull request merged**